test/recipes/80-test_ssl_new.t

```diff
@@ -123,7 +124,8 @@ my %skip = (
   "25-cipher.cnf" => disabled("ec") || disabled("tls1_2"),
   "26-tls13_client_auth.cnf" => disabled("tls1_3") || ($no_ec && $no_dh),
   "29-dtls-sctp-label-bug.cnf" => disabled("sctp") || disabled("sock"),
-  "31-quic.cnf" => $no_quic || $no_ec
+  "31-quic.cnf" => $no_quic || $no_ec,
+  "32-compressed-certificate.cnf" => disabled("comp") || disabled("tls1_3"),
```

Comment:
Don't we at least need one available compression algorithm?

Reply:
Yes and no. This test doesn't explicitly check for the use of compressed certificates (there are other tests that check the handshake messages and extensions). This one is testing that connections still succeed with the compression options regardless of whether compression is available.